skimage/draw/tests/test_random_shapes.py

```diff
@@ -135,9 +136,137 @@ def test_random_shapes_is_reproducible_with_seed():
     assert all(other == labels[0] for other in labels[1:])
 
 
-def test_generates_white_image_when_intensity_range_255():
-    image, labels = random_shapes((128, 128), max_shapes=3,
-                                  intensity_range=((255, 255),),
+def test_throws_when_intensity_range_255():
```

Comment:
`test_throws_when_intensity_range_255` -> `test_throws_when_intensity_range_equals_background` or something. Also, please specify the background value explicitly in the function call below.